py/kubeflow/kubeflow/ci/kfctl_go_test_utils.py

```diff
@@ -240,10 +240,17 @@ def kfctl_deploy_kubeflow(app_path, project, use_basic_auth, use_istio, config_p
 
   # TODO(jlewi): When we switch to KfDef v1beta1 this logic will need to change because
   # use_base_auth will move into the plugin spec
-  use_basic_auth = config_spec["spec"].get("useBasicAuth", False)
+  gcp_plugin = None
+  for plugin in config_spec["spec"]["plugins"]:
+    if plugin["kind"] == "KfGcpPlugin":
+      gcp_plugin = plugin
+      break
+  if gcp_plugin is None:
```

Comment:
We shouldn't assume gcp_plugin is set because we have non GCP configs.

Above default gcp_plugin to {}
and then the rest of the code should work.

Reply:
done